panel/_templates/basic_login.html

```diff
@@ -2,7 +2,11 @@
 <!DOCTYPE html>
 <html>
 <head>
```

Comment:
A use does not care about the type of authentication applied. But its nice to know you are on a login page.